offchain/dusd-ui/flake-module.nix

```diff
@@ -67,6 +94,7 @@
               cp -r ${ui}/lib/node_modules/dusd-ui/build/* $out/
               cp -r ${self'.packages."offchain:dusd-browser"}/dist/* $out/assets/scripts
               cp -r ${font-awesome-sprites}/*.svg $out/assets/images
+              cp -f ${optimized-images}/*.{jxl,png,webp} $out/assets/images
```

Comment:
We are copying files around all the time, having them multiple times in the nix store. Can this be a symlink?

Reply:
@marijanp great point, now fixed

Reply:
@marijanp symlink will not work in prod https://636d493b45b2ff125a83dbca--dusd-ui.netlify.app/, once I switched back to copying files, it works https://636d534a201fc923122f0812--dusd-ui.netlify.app/

Reply:
@marijanp I don't remember who said it, but I had read on the IRC that `cp` is preferred over symlinks the Nix store can better optimize itself not having to follow the indirection.